apps/docs/src/index.data.ts

```diff
@@ -64,7 +64,10 @@ export default defineLoader({
      */
     const getImplementedStatus = (componentName: string) => {
       const implemented = watchedFiles.some((file) => file.endsWith(`${componentName}.stories.ts`));
-      return { implemented, href: implemented ? `/development/${componentName}` : undefined };
+      return {
+        implemented,
+        href: implemented ? `/development/components/${componentName}` : undefined,
```

Comment:
we need to swap out OnyxRadioButton with OnyxRadioButtonGroup for this tile
<img width="725" alt="image" src="https://github.com/SchwarzIT/onyx/assets/151019977/60a1209e-5b62-46ec-82cc-811379f95af6">

because right now when you click it you are directed to 

http://localhost:5173/development/components/OnyxRadioButton.html 

-> hidden support component.

Reply:
Done